CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+- Delete the marketplace-operator deployment with `oc delete deployment openshift-marketplace -n marketplace-operator`.
+- Install [Operator-SDK](https://github.com/operator-framework/operator-sdk).
+- `operator-sdk up local --namespace=openshift-marketplace --kubeconfig=<path-to-kubeconfig-file>` to complie the marketplace-operator and start the operator in your dev environment. 
+
+### Testing changes to RBAC policy
+- If there's any modification in existing RBAC poilicies, the modifications need to be tested using a separate deployment. Easiest way to do this is to do a local `operator-sdk build` using [this dockerfile](./Dockerfile) and then `docker push` the image to a registry. Replace the [image here](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/operator.yaml#L23) with the newly built image, then do a `oc apply -f deploy/operator.yaml` to reinstall the operator.  
+
+## Marketplace-operator on vanilla Kubernetes
+
+- To set up marketplace-operator on vanilla Kubernetes, [install Operator-Lifecycle-Manager(OLM)](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md) in a Kubernetes Cluster. 
+- `cd` into the marketplace-operator repository and install the marketplace-operator using `kubectl apply -f deploy/upstream`.
+- To delete the marketplace-operator deployment use `oc delete deployment marketplace`. 
+
+### Testing changes to RBAC policy
+- If there's any modification in existing RBAC poilicies, the modifications need to be tested using a seperate deployment. Easiest way to do this is to do a local `operator-sdk build` [this dockerfile](./Dockerfile) and then `docker push` the image to a registry. Replace the [image here](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/upstream/07_operator.yaml#L28) with the newly built image, then do a `kubectl apply -f deploy/upstream/07_operator.yaml`.
+
+## Reporting bugs and creating issues
+
+If any part of the operator-marketplace project has bugs or documentation mistakes, please let us know by [opening an issue](https://github.com/operator-framework/operator-marketplace/issues/new) or a PR. 
+
+## Contribution flow
+
+This is an outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where to base the contribution. This is usually master.
+- Make commits of logical units.
+- Make sure commit messages are in the proper format (see below).
+- Push changes in a topic branch to a personal fork of the repository.
+- To make sure that your topic branch is in sync with the remote master branch, follow a rebase workflow.
+- Submit a pull request to operator-framework/operator-marketplace.
+- The PR must receive a LGTM from two maintainers of the operator-marketplace project.
```

Comment:
It is one /lgtm and one /approve from the maintainers. We try to give two /lgtm but there is nothing in the CI process that looks for that.